ingestion/functions/common/parsing_lib.py

```diff
@@ -54,6 +58,20 @@ def safe_int(x):
         return None
 
 
+def geocode_country(two_letter_iso_code):
+    two_letter_iso_code = two_letter_iso_code.upper()
+    try:
+        lat, lon, country = GEOCODING_COUNTRIES[two_letter_iso_code]
+        return {
+            "country": country,
+            "geoResolution": "Country",
+            "name": country,
+            "geometry": {"latitude": lat, "longitude": lon}
+        }
+    except KeyError:
+        return None
```

Comment:
I'd log the exception here too. Are there cases in which we expect the `two_letter_iso_code` to not be in `GEOCODING_COUNTRIES`?

Reply:
Could happen, we've seen for example that map box and google disagree on whether to only use two-letter country codes or also two-letter region codes for autonomous regions (e.g. `gi` for Gibraltar).